examples/util/tflite/TFliteModelImporter.js

```diff
@@ -133,7 +133,43 @@ class TFliteModelImporter {
     }, new Float32Array(tensor));
   }
 
-  _addOpsAndParams() {
+  async * layerIterator(inputTensors) {
+    const graph = this._rawModel.subgraphs(0);
+    const operatorsLength = graph.operatorsLength();
+    for (let lastNode = 0; lastNode < operatorsLength; ++lastNode) {
+      this._tensorIds = [];
+      this._operands = [];
+      this._operandIndex = 0;
+      if (this._backend !== 'WebML' && this._compilation) {
+        this._compilation._preparedModel._deleteAll();
+      }
+
+      this._model = await this._nn.createModel({backend: this._backend});
+      this._addTensorOperands();
+      lastNode = this._addOpsAndParams(lastNode);
+
+      const operator = graph.operators(lastNode);
+      const opCode = this._rawModel.operatorCodes(operator.opcodeIndex()).builtinCode();
+      const opcodeName = tflite.BuiltinOperator[opCode];
+
+      const inputs = Array.from(graph.operators(0).inputsArray());
+      const outputs = Array.from(operator.outputsArray());
+      this._model.identifyInputsAndOutputs(inputs, outputs);
+
+      await this._model.finish();
+      this._compilation = await this._model.createCompilation();
+      this._compilation.setPreference(getPreferCode(this._backend, this._prefer));
+      await this._compilation.finish();
+      this._execution = await this._compilation.createExecution();
+
+      const outputSize = this._operandTypes[outputs[0]].dimensions.reduce((a,b)=>a*b);
```

Comment:
There is no `_operandTypes` in TFliteModelImporter class. Could you please double check?